zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/IdentitySetupInitializeProcessor.java

```diff
@@ -239,6 +239,11 @@ final var record =
   private void addAllPermissions(final long roleKey) {
 
     for (final AuthorizationResourceType resourceType : AuthorizationResourceType.values()) {
+      if (resourceType == AuthorizationResourceType.UNSPECIFIED) {
```

Comment:
❓ Why ? We do have the same for `ownerType`

Reply:
We only want to create valid permissions. `UNSPECIFIED` exists so no one accidentally adds unintended permissions by us forgetting to set a resource type somewhere.

Reply:
Got it